.travis.yml

```diff
@@ -91,11 +91,20 @@ before_install:
     - pwd
     - mkdir /tmp/cozy-desktop
     - ln -s /tmp/cozy-desktop tmp
+    - npm install -g yarn
```

Comment:
I read that it installed the dependencies with yarn, but that it didn't install yarn itself.
I looked for a blog post about the latter, but couldn't find any...